docs/reference/migration/migrate_3_0.asciidoc

```diff
@@ -713,3 +713,7 @@ and `no` for disabling are no longer supported.
 === Term vectors
 
 The term vectors APIs no longer persist unmapped fields in the mappings.
+=== Replace use of Content-Type header in _cat API in favour of the Accept header
```

Comment:
Can you put this under a breaking cat API changes section (see the similar breaking REST API changes section)?